packages/devtools_app_shared/lib/src/service/isolate_manager.dart

```diff
@@ -237,6 +245,32 @@ final class IsolateManager with DisposerMixin {
     _isolateRunnableCompleters.clear();
   }
 
+  /// Resumes the isolate by calling [DdsExtension.readyToResume].
+  ///
+  /// CAUTION: This should only be used for a tool-initiated resume, not a user-
+  /// initiated resume. See:
+  ///  https://github.com/dart-lang/sdk/commit/5536951738ba599d96e075b7140e52b28e233
+  Future<void> resumeIsolate(IsolateRef isolateRef) async {
+    if (isolateRef.id == null || _service == null) return;
+    final isolateId = isolateRef.id!;
+    try {
+      await _readyToResume(isolateId);
+    } catch (error) {
+      _log.warning(error);
+    }
+  }
+
+  Future<void> _readyToResume(String isolateId) async {
+    final service = _service!;
+    try {
+      await service.readyToResume(isolateId);
+    } on UnimplementedError {
```

Comment:
Does this actually throw `UnimplementedError` on this end? I would assume this would actually result in an `RPCError` with code `kMethodNotFound`.

Reply:
Just verified that is does throw `UnimplementedError` (for both web and non-web apps)